res/core/messages.xml

```diff
@@ -8441,4 +8441,15 @@
     <text locale="en">"$unit($unit) in $region($region): '$order($command)' - Heroes cannot recruit."</text>
   </message>
 
+  <message name="dragon_growl" section="mail">
+    <type>
+      <arg name="dragon" type="unit"/>
+      <arg name="number" type="int"/>
+      <arg name="target" type="region"/>
+      <arg name="choice" type="int"/>
+    </type>
+    <text locale="de">"$unit($dragon): \"$if($eq($choice,0), "Groaaaam... ", "")$if($eq($choice,1), "Chrrrr... ", "")$if($eq($choice,2), "Roooarrr... ", "")$if($eq($choice,3), "Tschrrrk... ", "")$if($eq($choice,4), "Shhhhhh... ", "")$if($eq($number,1), "Ich rieche", "Wir riechen") etwas in $region($target)\"."</text>
```

Comment:
Im Code gefunden: Die Funktion heisst `$localize`, und ist in `eval_localize` implementiert.